### PR TITLE
chore(scaletest/templates/scaletest-runner): fix dashboard command invocation, autoscale provisioners

### PR DESCRIPTION
**Pull request by johnstcn**
* add --retries on kubectl cp
* remove --count parameter to scaletest dashboard
* scale provisioners up and down

### CODE REVIEW

File: scaletest/templates/scaletest-runner/scripts/lib.sh [hunk 1]
```diff
@@ -271,10 +271,12 @@ fetch_coder_full() {
 		exit 1
 	fi
 	log "Fetching full Coder binary from ${pod}"
+	# We need --retries due to https://github.com/kubernetes/kubernetes/issues/60140 :(
 	maybedryrun "${DRY_RUN}" kubectl \
 		--namespace "${namespace}" \
 		cp \
 		--container coder \
+		--retries 3 \
```
Comment (mafredri) [inline scaletest/templates/scaletest-runner/scripts/lib.sh:279]: Only 3? 😅 Maybe we'll go a bit nuttier? 😉

```suggestion
		--retries 10 \
```

File: scaletest/templates/scaletest-runner/scripts/prepare.sh [hunk 2]
```diff
@@ -51,3 +51,9 @@ log "Cleaning up from previous runs (if applicable)..."
 "${SCRIPTS_DIR}/cleanup.sh" "prepare"
 
 log "Preparation complete!"
+
+log "Scaling up provisioners to ${SCALETEST_PARAM_CREATE_CONCURRENCY}..."
+maybedryrun "$DRY_RUN" kubectl scale deployment/coder-provisioner \
+	--replicas "${SCALETEST_PARAM_CREATE_CONCURRENCY}"
```
Comment (mafredri) [inline scaletest/templates/scaletest-runner/scripts/prepare.sh:57]: This is dope! 😍

**Review by mafredri — APPROVED** [review]

File: scaletest/templates/scaletest-runner/scripts/lib.sh (hunk 1 above)
Comment (johnstcn) [reply scaletest/templates/scaletest-runner/scripts/lib.sh:279]: lol the linked github issue has folks setting 999 retries so 10 is relatively tame by comparison

File: scaletest/templates/scaletest-runner/scripts/prepare.sh (hunk 2 above)
Comment (johnstcn) [reply scaletest/templates/scaletest-runner/scripts/prepare.sh:57]: actually I may need to handle concurrency=0 🤔

Comment (johnstcn) [reply scaletest/templates/scaletest-runner/scripts/prepare.sh]: OK, if `SCALETEST_PARAM_CREATE_CONCURRENCY` is 0, I'm defaulting to 10 provisioners.

**Pull request merged**